crypto/pkcs12/p12_attr.c

```diff
@@ -119,3 +119,8 @@ PKCS12_SAFEBAG_get0_attrs(const PKCS12_SAFEBAG *bag)
 {
     return bag->attrib;
 }
+
+void PKCS12_SAFEBAG_set_attrs(const STACK_OF(X509_ATTRIBUTE) *attrs, PKCS12_SAFEBAG *bag)
```

Comment:
The `PKCS12_SAFEBAG` parameter should be first. This probably should be called `PKCS12_SAFEBAG_set0_attrs` (the 0 indicates that the "ownership" of the attributes passes to the safe bag)